pandas/tests/series/test_constructors.py

```diff
@@ -95,6 +95,14 @@ def test_scalar_conversion(self):
         assert float(Series([1.0])) == 1.0
         assert int(Series([1.0])) == 1
 
+    @pytest.mark.parametrize("scalar", (Interval(0, 1), Period("2019Q1", freq="Q")))
```

Comment:
can you create a fixture of the ea_scalars and use it here: https://github.com/pandas-dev/pandas/blob/master/pandas/tests/frame/test_constructors.py#L738 and in the new test you are adding

Reply:
there are also similar tests for a dict of a scalar in both series/frame which could be updated to use the fixture as well

Reply:
> can you create a fixture of the ea_scalars and use it here: https://github.com/pandas-dev/pandas/blob/master/pandas/tests/frame/test_constructors.py#L738 and in the new test you are adding

Done